.jenkins/pytorch/test.sh

```diff
@@ -750,19 +788,39 @@ elif [[ "${TEST_CONFIG}" == *dynamo* && "${SHARD_NUMBER}" == 2 && $NUM_TEST_SHAR
   install_filelock
   install_triton
   test_dynamo_shard 2
+elif [[ "${TEST_CONFIG}" == *inductor_huggingface_perf* ]]; then
```

Comment:
Can we merge this `elif` with the following on and use `if` only for the last step to select test_inductor_huggingface or test_inductor_huggingface_perf?

Reply:
Done, thanks for the suggestion.